src/packager.ts

```diff
@@ -185,8 +199,11 @@ const concatAllObjects = async (
     for (const { Key: key } of objects) {
       const { Body: data } = await safeSend(client, bucket, key);
 
-      await fs.appendFile(outputFile, data);
-      await fs.appendFile(outputFile, "\n");
+      if (data) {
+        const stringData = await data.transformToString();
```

Comment:
Any reason why `data` now needs to be transformed to a string before writing it to file?